### PR TITLE
Modified the testcase to use avocado library

### PR DESCRIPTION
**Pull request by shirishaganta**
Signed-off-by: shirisha Ganta <shiganta@in.ibm.com>

### CODE REVIEW

**Comment by shirishaganta** [comment]
After Patch:
[root@ltcfleet2-lp18 ras]# avocado run packages.py
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : b3affacf1b0cf4a6aae468f68813552c37501bd3
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2020-06-15T15.58-b3affac/job.log
 (1/1) packages.py:Package_check.test: PASS (1.28 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2020-06-15T15.58-b3affac/results.html
JOB TIME   : 1.32 s
[root@ltcfleet2-lp18 ras]#

File: ras/packages.py [hunk 1]
```diff
@@ -34,40 +33,16 @@ def setUp(self):
 
     def test(self):
         dist = distro.detect()
-        is_fail = 0
```
Comment (harish-24) [inline ras/packages.py:37]: why is this removed? How does the test fail?

**Comment by shirishaganta** [comment]
[root@ltczz256-lp2 ras]# avocado run packages.py
/usr/local/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : fe84e73f0ae61ca6eee075d3f13a46bb4ba2a056
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2020-06-16T00.29-fe84e73/job.log
 (1/1) packages.py:Package_check.test:  PASS (1.52 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2020-06-16T00.29-fe84e73/results.html
JOB TIME   : 1.68 s
[root@ltczz256-lp2 ras]#

File: ras/packages.py [hunk 2]
```diff
@@ -39,35 +38,15 @@ def test(self):
...
-                if self.sm.check_installed(package):
-                    self.log.info("%s package is installed" % package)
-                else:
-                    self.log.info("%s package is not installed" % package)
+        for package in self.packages:
+            if self.sm.check_installed(package):
+                self.log.info("%s package is installed" % package)
+            else:
+                self.log.info("%s package is not installed" % package)
+                is_fail += 1
```
Comment (harish-24) [inline ras/packages.py:50]: Nit pick: instead of incrementing fails, suggest you to collect failed packages. If you do not intend to do that, you can change this to a bool flag.

File: ras/packages.py [hunk 3]
```diff
@@ -34,40 +33,19 @@ def setUp(self):
...
-                    self.log.info("%s package is not installed" % package)
-        if is_fail >= 1:
-            self.fail("%s package(s) not installed by default" % is_fail)
+        not_installed_list = []
+        for package in self.packages:
+            if self.sm.check_installed(package):
+                self.log.info("%s package is installed" % package)
+            else:
+                not_installed_list.append(package)
+        if (len(not_installed_list) >= 1):
```
Comment (harish-24) [inline ras/packages.py:50]: `if not_installed_list:`
